File: java/027_darts/Darts.java
class Darts {
    int score(double x, double y) {
        double result = Math.sqrt(x * x + y * y);
        if(result <= 1.0) return 10;
        if(result <= 5.0) return 5;
        if(result <= 10.0) return 1;
        return 0;
    }
}

File: csharp/ex_086_darts/Darts.cs
using System;

public static class Darts
{
    public static int Score(double x, double y)
        => Math.Sqrt(x * x + y * y) switch {
            <= 1.0  => 10,
            <= 5.0  => 5,
            <= 10.0 => 1,
            _       => 0
        };
}